samples/subsys/usb/uvc/boards/frdm_mcxn947_mcxn947_cpu0.conf

```diff
@@ -0,0 +1,5 @@
+# Temporary workaround a build-time incompatibility
```

Comment:
What is the workaround? The whole file?

Reply:
Not the debug logs, but all 3 other options are.

Ideally, none of that would require any overlay, as this test does not have any hardware dependency except USB.

I can remove this last line and will make the comment more obvious.
Generally, remove the Kconfig related to comfort of testing: quite subjective!

Reply:
Souunds good, thank you for the clarification.

Reply:
The end goal is to allow using the built-in video, so removed that overlay in the meantime that https://github.com/zephyrproject-rtos/zephyr/issues/82755 is solved.